src/web/components/MainMedia.tsx

```diff
@@ -61,6 +61,7 @@ function renderElement(
     hideCaption?: boolean,
     adTargeting?: AdTargeting,
     starRating?: number,
+    host?: string,
```

Comment:
Can we move these live changes out into a separate PR. These looks like actual code changes to how thing function rather than being related to the test.

If they _are_ only needed for the test then we should question if we want to add these props. As a rule it is never great to add props purely for testing. Each prop that gets added comes at a cost and ideally we should limit them as much as possible.